src/utils/git.ts

```diff
@@ -34,3 +36,35 @@ export function getCurrentGitBranch(): string | undefined {
 
   return currentBranch
 }
+
+export function getGitExtensionAPI(): GitExtensionAPI {
+  const gitExtensionId = 'vscode.git'
+  const gitExtension = vscode.extensions.getExtension<GitExtension>(gitExtensionId)
+
+  if (gitExtension === undefined) {
+    throw new Error(`Could not load VSCode Git extension with id '${gitExtensionId}'`)
```

Comment:
Given that the chances that this ever breaks are _very_ low, we still should handle the case of it throwing.

It could be either falling back to httpd or just notifying the user with an error message of what happened.